mobile/src/calendar/days/tile/tile-indicator.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { styles, colors } from '../../styles';
+
+type RegisteredStyles =  { [propName: string]: any };
+
+interface TileIndicatorStyles {
+    left: RegisteredStyles;
+    all: RegisteredStyles;
+}
+
+interface TileIndicatorProps {
+    value: number;
+    leftColor: string;
+    allColor: string;
+}
+
+export class TileIndicator extends Component<TileIndicatorProps> {
+    private styles: TileIndicatorStyles;
+    private readonly allDays = 100;
+
+    public render() {
+        this.calculateStyles();
+
+        return (
+            <View style={styles.tileIndicator}>
+                <View style={this.styles.left} />
+                <View style={this.styles.all} />
+            </View>
+        );
+    }
+
+    private calculateStyles() {
+        this.styles = StyleSheet.create({
```

Comment:
couldn't you make it work without this magic? :)

Reply:
Removed